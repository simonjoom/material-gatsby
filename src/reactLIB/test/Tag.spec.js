/* global describe, it, expect */

import React from 'react';
import { shallow } from 'enzyme';
import Tag from '../src/Tag';

const child = 'hey';
const wrapper = shallow(<Tag>{child}</Tag>);

describe('Tabs', () => {
  test('renders', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('should create list of Tab itemt', () => {
    expect(wrapper.contains(child)).toEqual(true);
  });
});
