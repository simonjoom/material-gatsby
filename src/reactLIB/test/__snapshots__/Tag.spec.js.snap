// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs renders 1`] = `
<Chip
  close={true}
>
  hey
</Chip>
`;
